contrib_bots/lib/github_detail.md

```diff
@@ -0,0 +1,27 @@
+# GitHub detail bot
+This bot links and details issues and pull requests.
+To use it username mention the bot then type an id:
```

Comment:
The bot reacts to it's username being mentioned in chat, that means @**<username>** which is how zulip's autocomplete mentions are formatted.

I did this because the intuitive way to mention a user/bot is to type @ then start typing in their name. It's also easier because you only have to type @Github... or something like that then the mention will be auto-completed.